python/sdk/merlin/model.py

```diff
@@ -1045,13 +1045,18 @@ def deploy(self, environment_name: str = None,
         if resource_request is None:
             env_api = EnvironmentApi(self._api_client)
             env_list = env_api.environments_get()
+            
             for env in env_list:
                 if env.name == target_env_name:
                     resource_request = ResourceRequest(
                         env.default_resource_request.min_replica,
                         env.default_resource_request.max_replica,
                         env.default_resource_request.cpu_request,
-                        env.default_resource_request.memory_request)
+                        env.default_resource_request.memory_request,
+                        env.default_resource_request.gpu_request if env.default_resource_request.gpu_request else None,
+                        env.default_resource_request.gpu_node_selector.get("cloud.google.com/gke-accelerator", None) if env.default_resource_request.gpu_resource_type else None
```

Comment:
We don't need this because we will never set GPU value in default_resource_request.